lib/internal/fs/watchers.js

```diff
@@ -309,11 +316,15 @@ async function* watch(filename, options = kEmptyObject) {
     persistent = true,
     recursive = false,
     encoding = 'utf8',
+    maxQueue = 2048,
+    overflow = 'swallow',
     signal,
   } = options;
 
   validateBoolean(persistent, 'options.persistent');
   validateBoolean(recursive, 'options.recursive');
+  validateInteger(maxQueue, 'options.maxQueue');
```

Comment:
Can you add a new test that includes basic checks verifying passing in wrong type of `maxQueue` and `overflow` would result in error with `{ code: 'ERR_INVALID_ARG_TYPE' }`

Reply:
Done in 23374430d28344715067bd94eb3d3356aded574e